sync-s3.sh

```diff
@@ -59,3 +59,17 @@ aws s3 sync . "s3://${s3_bucket}" --delete --follow-symlinks --no-progress --exc
 
 echo "===> Syncing file lists with the S3 bucket"
 aws s3 sync . "s3://${s3_bucket}" --delete --follow-symlinks --no-progress --exclude "*" --include "*/list.*"
+
+echo "===> Invalidating CloudFront cache"
+# Invalidate only the files that might change in-place when new binaries are added.
+# NOTE: Invalidation paths allow wildcards only as the last character. When used at
+# any other position, AWS will not report an error but will also not invalidate it.
+# NOTE: The code below assumes that paths do not contain whitespace.
+aws cloudfront create-invalidation \
+    --distribution-id E1O6GT57WUFUHD \
```

Comment:
Where does the ID come from? Is it used multiple times (in other files)?

Reply:
It's the ID CloudFront uses to identify the "distribution", i.e. the project you create there to cache and serve stuff from the S3 bucket. We have only one distribution and it's permanent so this should not change unless we scrap it all and configure CloudFront from scratch.

It's not a secret key of any kind if that's what you're worried about. Secrets are in https://github.com/ethereum/solc-bin/blob/6b4ae0223301ecba154b963e0c318308f52a9cee/.github/workflows/s3-mirror.yml#L37-L41 and they come from repo config.

This script is the only place we're using it currently. But you have a point, I see that I did not even hard-code the bucket IDs so having this ID here is a bit out of place. I'll move it to `s3-mirror.yml`.

Reply:
Done.